### DISCUSSION TITLE
Solid Start cannot be deployed with Solid Auth and Auth.js

### DISCUSSION THREAD

**Discussion by NathanPip** [vercel/vercel#9423] [Help]
Hi,

I was testing out Solid Start with Auth.js and ran into an issue when trying to deply to Vercel.

`Error: Builder returned invalid routes: ["Route at index 5 has invalid src regular expression \"^/api/auth/(?<solid-auth>.*)$\"."]`

I have no special configurations and just setup basic authentication with Github and the post build scripts I have setup run before this error occurs so I'm assuming it's a platform issue. Just wondering if anybody has had a similar issue using these packages.

Thank you!

**Comment by swarnava** [comment]
Hi, Do you have a [minimal repository](https://stackoverflow.com/help/minimal-reproducible-example) which we can look into?
